src/sumo_gym/utils/xml_utils.py

```diff
@@ -98,3 +158,32 @@ def _parse_network_xml(network_file_path: str):
             edge_count += 1
 
     return vertices, demand, edge_id_map, edges
+
+
+def cli_main():
+    """ CLI commands for testing """
+
+    if len(sys.argv) == 1:
+        return
+
+    # python3 xml_utils.py decode_xml_fmp {net_xml_file_path}
+    if sys.argv[1] == "decode_xml_fmp":
+        decode_xml_fmp(sys.argv[2])
+
+
+
+# entry point to CLI
+if __name__ == "__main__":
+
+    cli_main()
```

Comment:
probably we don't need main function in public lib, please clean this local testing, you can stash it and pop back later!